95% of plugins fail to install on Windows when using GUI

Whenever trying to install certain plugins, the plugin manager GUI, it fails with the one of two errors:
---
### 1.  `can't find repository <plugin here>` (e.g. `language_go:0.1`)


Example with plugin `language_go`
```
09/09/23 18:19:20 [ERROR] error running C:\Users\quinten\.config\lite-xl\plugins\plugin_manager\lpm.x86_64-windows.exe install language_go:0.1 --json --mod-version=3 --quiet --progress --userdir=C:\Users\quinten\.config\lite-xl --datadir=C:\Users\quinten\AppData\Local\Programs\Lite XL\data --binary=C:\Users\quinten\AppData\Local\Programs\Lite XL\lite-xl.exe --assume-yes: {"warnings":[],"actions":[],"error":"src/lpm.lua:1006: can't find repository language_go:0.1"}
 at ...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:125

stack traceback:
...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:125: in function <...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:98>
[C]: in function 'xpcall'
...uinten\AppData\Local\Programs\Lite XL\data/core/init.lua:1225: in function 'core.try'
(...tail calls...)
```

I can reproduce this error within the CLI by trying `.\lpm.x86_64-windows.exe install language_go:0.1`. This gives the message: `src/lpm.lua:1006: can't find repository language_go:0.1`. 
I _can_ however install `language_go` without the version number attached ( `.\lpm.x86_64-windows.exe install language_go`). This gives no error messages whatsoever.

---
### 2. `only repositories with http and file transports are supported`

Example with plugin ipc:
```
09/09/23 19:26:26 [ERROR] error running C:\Users\quinten\.config\lite-xl\plugins\plugin_manager\lpm.x86_64-windows.exe install ipc:0.3 --json --mod-version=3 --quiet --progress --userdir=C:\Users\quinten\.config\lite-xl --datadir=C:\Users\quinten\AppData\Local\Programs\Lite XL\data --binary=C:\Users\quinten\AppData\Local\Programs\Lite XL\lite-xl.exe --assume-yes: {"actions":[],"warnings":[],"error":"src/lpm.lua:965: only repositories with http and file transports are supported (ipc)"}
 at ...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:125

stack traceback:
...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:125: in function <...\quinten\.config\lite-xl/plugins\plugin_manager/init.lua:98>
[C]: in function 'xpcall'
...uinten\AppData\Local\Programs\Lite XL\data/core/init.lua:1225: in function 'core.try'
(...tail calls...)
```

Again, this error is also reproducible using the CLI, and the same "fix" applies.
`.\lpm.x86_64-windows.exe install ipc:0.3` -> `src/lpm.lua:965: only repositories with http and file transports are supported (ipc)`
`.\lpm.x86_64-windows.exe install ipc` -> `Installing singleton plugin ipc.`

---
I have tried purging and updating the standard repositories. Lite-XL (version 2.1.1) was just freshly installed, with the standard `.config` folder wiped completely beforehand. 
Video demonstrating the error in both CLI and GUI: https://streamable.com/gw4q1d

Was unsure whether or not to bundle these two errors into a single issue, but since they both seem to originate from the same problem I figured they are probably related.

--- a/src/lpm.lua
+++ b/src/lpm.lua
@@ -1648,7 +1648,7 @@ local function lpm_lite_xl_list()
 end
 
 local function is_argument_repo(arg)
-  return arg:find("^http") or arg:find("[%.\\/]")
+  return arg:find("^http") or arg:find("[\\/]") or arg == "."
 end
 
 local function lpm_lite_xl_run(version, ...)

